<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="ADMIN-PC" failures="1" tests="5" name="TestNG_Test2" time="0.211" errors="0" timestamp="2020-12-02T16:29:15 PST">
  <testcase classname="sample_TestNG.SampleTestNGtest1" name="close1" time="0.015"/>
  <testcase classname="sample_TestNG.SampleTestNGtest1" name="login1" time="0.007"/>
  <testcase classname="sample_TestNG.SampleTestNGtest1" name="setup1" time="0.003"/>
  <testcase classname="sample_TestNG.SampleTestNGtest" name="setup" time="0.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at sample_TestNG.SampleTestNGtest.setup(SampleTestNGtest.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- setup -->
  <testcase classname="sample_TestNG.SampleTestNGtest" name="close" time="0.005"/>
  <testcase classname="sample_TestNG.SampleTestNGtest" name="login" time="0.0">
    <ignored/>
  </testcase> <!-- login -->
</testsuite> <!-- TestNG_Test2 -->
